avaframe/in3Utils/geoTrans.py

```diff
@@ -278,20 +282,90 @@ def remeshData(rasterDict, cellSizeNew, remeshOption="griddata", interpMethod="c
+        of specified size, default resampling option is set to cubic
+
+        Parameters
+        -------------
+        rasterFile: pathlib Path
+            path to file with raster data options asci or tif
+        cellSizeNew: float
+            desired spatial resolution of new raster dataset, cellsize
+        larger: Boolean
+            if true (default) output grid is at least as big as the input
+
+        Returns
+        ---------
+        remeshedRaster: dict
+            dictionary with header and rasterdata with new cellSize
+     """
+    # open raster data
+    src = rasterio.open(rasterFile, 'r')
+
+    # First part is to be able to replicate our own remeshing
+    header = rU.getHeaderFromRaster(src)
+    _, _, width, height = makeCoordGridFromHeader(header, cellSizeNew=cellSizeNew, larger=larger)
+
+    targetTransform = rasterio.transform.from_origin(header['xllcenter'] - cellSizeNew / 2.0,
+                                                     (header['yllcenter'] - cellSizeNew / 2.0) + height *
+                                                     cellSizeNew,
+                                                     cellSizeNew,
+                                                     cellSizeNew)
+
+    # Check if crs is none, as warp.reproject NEEDS a valid crs
+    # If none is available, set any. The original is taken again when
+    # the header is reset lower down
+    # TODO: require a default crs in config
+    if src.crs is None:
+        srcCrs = rasterio.crs.CRS.from_epsg(31287)
+    else:
+        srcCrs = src.crs
+
+    data, transform = rasterio.warp.reproject(source=src.read(),
+                                              destination=np.empty((src.count, height,
+                                                                    width)) * np.nan,
+                                              src_transform=src.transform,
+                                              dst_transform=targetTransform,
+                                              src_crs=srcCrs,
+                                              dst_crs=srcCrs,
+                                              dst_nodata=src.nodata,
+                                              resampling=Resampling.cubic)
+
+    data = np.where(data == src.nodata, np.nan, data)
+    # create header of resampled data
+    # set new header
+    headerRemeshed = rU.getHeaderFromRaster(src)
+    src.close()
+
+    headerRemeshed["cellsize"] = transform[0]
+    headerRemeshed["transform"] = transform
+    headerRemeshed["ncols"] = data[0].shape[1]
+    headerRemeshed["nrows"] = data[0].shape[0]
+    headerRemeshed["nodata_value"] = 'nan'
+
+    # create remeshed raster dictionary
+    remeshedRaster = {"rasterData": data[0], "header": headerRemeshed}
+
+    return remeshedRaster
+
+def remeshRaster(rasterFile, cfgSim, typeIndicator="DEM", onlySearch=False, legacy=False):
```

Comment:
update docstring that now based on rasterio resampling not griddata maybe hint to legacy option? In line 374